ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java

```diff
@@ -33,9 +33,12 @@ public CCDIndividual to(Individual individual) {
         individual.getRepresentative()
             .ifPresent(representative -> builder.representative(representativeMapper.to(representative)));
 
+        if (individual.getDateOfBirth() != null) {
+            builder.dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE));
```

Comment:
DOB should be optional when nullable.

Reply:
Also create test case for to prevent failures in the future

Reply:
sure. Will do